samples/tutorial/Tutorial2.md

```diff
@@ -32,51 +12,34 @@ Start from the implementation of `tutorial-1-complete` if you're skipping ahead.
 
 Let's add a second screen and workflow so we have somewhere to land after we log in. Our next screen will be a list of "todo" items, as todo apps are the best apps.
 
-Create a new screen/`LayoutRunner` pair called `TodoList`:
-
-Add the provided `TodoListViewBinding` from `tutorial-views` as a subview to the newly created layout runner:
+Create a new screen called `TodoListScreen`, using the provided `TodoListViewBinding` from `tutorial-views`. Note the extra setup of an adapter for the `todoList` `RecyclerView`.
 
 ```kotlin
-/**
- * This should contain all data to display in the UI.
- *
- * It should also contain callbacks for any UI events, for example:
- * `val onButtonTapped: () -> Unit`.
- */
 data class TodoListScreen(
-  val username: String,
-  val todoTitles: List<String>,
-  val onTodoSelected: (Int) -> Unit,
-  val onBack: () -> Unit
-)
-
-class TodoListLayoutRunner(
-  /** From `todo_list_view.xml`. */
-  private val todoListBinding: TodoListViewBinding
-) : LayoutRunner<TodoListScreen> {
+  val TBD: String = ""
```

Comment:
Should we do same placeholder in Tutorial1 too? Seems nice that it compiles

Reply:
You're talking about this bit?

```
data class WelcomeScreen(
  // TODO: add properties needed to update WelcomeViewBinding
) : AndroidScreen<WelcomeScreen> {
```

I believe I made that one match what gets produced by the template, meant it as a signal that they're on exactly the right path.

Reply:
Okay, yep. If intentional all good!